src/AutoRest.CSharp/LowLevel/Generation/LowLevelExampleComposer.cs

```diff
@@ -0,0 +1,1011 @@
+            {
+                for (int i = 1; i < ClientInvocationChain.Count; i++)
+                {
+                    var factoryMethod = ClientInvocationChain[i];
+                    code += GetFactoryMethodCode(factoryMethod);
+                }
+            }
+            builder.Append(code).AppendLine(";");
+        }
+
+        private string MockClientConstructorParameterValues(IReadOnlyList<Parameter> parameters)
+        {
+            var parameterValues = new List<string>(parameters.Count);
+            foreach (var parameter in parameters)
+            {
+                if (parameter.Type.EqualsIgnoreNullable(UriType))
+                {
+                    parameterValues.Add("endpoint");
+                }
+                else if (parameter.Name.Equals("endpoint", StringComparison.OrdinalIgnoreCase))
+                {
+                    // sometimes the endpoint parameter cannot be generated as Uri type, best efforts to guesss it
+                    // see: https://github.com/Azure/autorest/issues/4571
+                    parameterValues.Add($"\"<{GetEndpoint()}>\"");
+                }
+                else if (parameter.Type.EqualsIgnoreNullable(KeyAuthType) || parameter.Type.EqualsIgnoreNullable(TokenAuthType))
+                {
+                    parameterValues.Add("credential");
+                }
+                else
+                {
+                    parameterValues.Add(MockParameterValue(parameter));
+                }
+            }
+            return string.Join(", ", parameterValues);
+        }
+
+        /// <summary>
+        /// Get the methods to be called to get the client, it should be like `Client(...).GetXXClient(..).GetYYClient(..)`.
+        /// It's composed of a constructor of non-subclient and a optional list of subclient factory methods.
+        /// </summary>
+        /// <returns></returns>
+        private static IReadOnlyList<MethodSignatureBase> GetClientInvocationChain(LowLevelClient client)
+        {
+            var callChain = new Stack<MethodSignatureBase>();
+            while (client.FactoryMethod != null)
+            {
+                callChain.Push(client.FactoryMethod.Signature);
+                if (client.ParentClient == null)
+                {
+                    break;
+                }
+
+                client = client.ParentClient;
+            }
+            callChain.Push(client.SecondaryConstructors.Where(c => c.Modifiers == MethodSignatureModifiers.Public).OrderBy(c => c.Parameters.Count).First());
+
+            return callChain.ToList();
+        }
+
+        private string GetFactoryMethodCode(MethodSignatureBase factoryMethod)
+        {
+            return $".{factoryMethod.Name}({MockParameterValues(factoryMethod.Parameters, true)})";
+        }
+
+        private static string GetEndpoint()
+        {
+            // TODO: is there a way to compose a data plane endpoint from swagger?
+            return "https://my-account-name.azure.com";
```

Comment:
Is it better to use 'https://my-service.azure.com'?  'my-account-name.azure.com' seems a little confuse that it is a resource account.